ui/apps/platform/src/Containers/Integrations/IntegrationForm/IntegrationForm.tsx

```diff
@@ -100,6 +103,16 @@ function IntegrationForm({
     initialValues,
     isEditable,
 }: IntegrationFormProps): ReactElement {
+    const history = useHistory();
+
+    const { isCentralCapabilityAvailable } = useCentralCapabilities();
+    const canUseCloudBackupIntegrations = isCentralCapabilityAvailable(
+        'centralCanUseCloudBackupIntegrations'
+    );
+    if (!canUseCloudBackupIntegrations && source === 'backups') {
```

Comment:
The condition reads backward to my eyes: the capability is relevant only for backups.
Ditto for IntegrationsListPage.